templates/components/common/navigation-list-alternate.html

```diff
@@ -0,0 +1,8 @@
+{{#if children}}
+    <a class="navPages-action navPages-action-depth-max has-subMenu is-root" href="{{url}}" data-collapsible="navPages-{{id}}">
+        {{name}} <i class="icon navPages-action-moreIcon" aria-hidden="true"><svg><use xlink:href="#icon-chevron-down" /></svg></i>
+    </a>
+    {{> components/common/navigation-dropdown}}
+{{else}}
+    <a class="navPages-action" href="{{url}}">{{name}}</a>
+{{/if}}
```

Comment:
haha, yes, it's been added to navigation-list-alternate.html.  was mixing up config.json.